_pytest/terminal.py

```diff
@@ -19,12 +19,39 @@
     EXIT_USAGEERROR, EXIT_NOTESTSCOLLECTED
 
 
+import argparse
+
+
+class MoreQuietAction(argparse.Action):
+    def __init__(self,
+                 option_strings,
+                 dest,
+                 default=None,
+                 required=False,
+                 help=None):
+        super(MoreQuietAction, self).__init__(
+            option_strings=option_strings,
+            dest=dest,
+            nargs=0,
+            default=default,
+            required=required,
+            help=help)
+
+    def __call__(self, parser, namespace, values, option_string=None):
+        new_count = getattr(namespace, self.dest, 0) - 1
+        setattr(namespace, self.dest, new_count)
+        # todo Deprecate config.quiet
+        namespace.quiet = getattr(namespace, 'quiet', 0) + 1
```

Comment:
Hmm do we really need to deprecate `namespace.quiet`? If all we need to do is to keep this line around, I wouldn't bother TBH.

Reply:
its random shitty data showing up in introspection

on top of that its half-dead zombie data and partially incorrect, it should go away

Reply:
We can't we remove it now? Is it because it might be used by thirdparty plugins?